src/localstream.js

```diff
@@ -209,25 +241,43 @@ const localstream = (function() {
 		console.log('(ReachSDK::localstream::initAudioVideo)');
 		if (!initAudioVideoProgress) {
 			initAudioVideoProgress=true;
+			_getAllVideoSources();
 
 			navigator.getMedia = getUserMedia;
 			mLocalStreamAudioVideo = document.createElement('AUDIOVIDEO');
 			mLocalStreamAudioVideo.muted= true;
-			localAudioVideoStreams.push(mLocalStreamAudioVideo);
+			//localAudioVideoStreams.push(mLocalStreamAudioVideo);
+
+			let audioSource;
+			let videoSource;
+
+			if(videoSources[currentVideoSource]){
+				videoSource = videoSources[currentVideoSource].deviceId;
+			}
+
```

Comment:
there is already an error callback

,
					(error) => {
						console.error('(ReachSDK::localstream::initAudioVideo::Error on webrtcLocalStream - webkitGetUserMedia : error=');
						console.dir(error);
						initAudioVideoProgress=false;

						if (callback) {
							callback(error);
						}
					}